safe-global/safe-wallet-web
# feat: Move on-chain sign messages to new tx flow

## What it solves

Part of #2067 

## How this PR fixes it

- Moves the On-Chain sign message modal to the new flow

## How to test it

1. Either connect to a safe app that uses on-chain messages or use the demo eip1271 app and connect via WC safe app
2. Go to Safe Apps settings and turn on "Always use on-chain signatures"
3. Trigger a sign message from the app
4. Observe the on-chain message in the new flow

## Screenshots
<img width="1139" alt="Screenshot 2023-06-15 at 14 51 24" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/9ca9640d-d51b-4b7e-bdc2-e0ef7ab24094">

## Checklist
